crates/oxc_prettier/src/lib.rs

```diff
@@ -105,6 +105,10 @@ impl<'a> Prettier<'a> {
         (len > n).then(|| self.stack[len - n - 1])
     }
 
+    fn find_ancestor(&self, predicate: impl Fn(AstKind<'a>) -> bool) -> Option<AstKind<'a>> {
+        self.stack.iter().rev().skip(2).find(|&&kind| predicate(kind)).copied()
```

Comment:
This `skip(2)` looks out of place for `find_ancestor `, probably need a comment?

Reply:
Thanks! This should be `skip(1)` for the current node. I also added a comment for this.